lib/types/index.js

```diff
@@ -18,3 +18,5 @@ exports.ObjectId = require('./objectid');
 exports.Map = require('./map');
 
 exports.Subdocument = require('./subdocument');
+
+exports.UUID = require('./uuid.js');
```

Comment:
I would just make this `exports.UUID = require('mongodb').BSON.UUID;`, the `uuid.js` file is unnecessary for several reasons.